src/ControlSystem/Tags.hpp

```diff
@@ -184,4 +168,59 @@ struct MeasurementTimescales : db::SimpleTag {
   }
 };
 }  // namespace Tags
+
+/// \ingroup ControlSystemGroup
+/// Holds all options for a single control system
+///
+/// This struct collects all the options for a given control system during
+/// option parsing. Then during initialization, the options can be retrieved via
+/// their public member names and assigned to their corresponding DataBox tags.
+template <size_t DerivOrder>
+struct OptionHolder {
+  struct Averager {
+    using type = ::Averager<DerivOrder>;
+    static constexpr Options::String help = {"Options for the averager."};
```

Comment:
Please describe these three options a bit in their help string. A help string that just repeats the name of the options is not very helpful ;) You don't even have to say "options for ..." since everything in the input file are options. Just give the user an idea what the averager etc is, and what they should keep in mind for setting its options (if any).